integration/skaffold/helper.go

```diff
@@ -262,6 +276,15 @@ func (b *RunBuilder) cmd(ctx context.Context) *exec.Cmd {
 	return cmd
 }
 
+func isCoreCommand(command string) bool {
+	switch command {
+	case "build", "debug", "delete", "deploy", "dev", "generate-pipeline", "render", "run":
```

Comment:
I'm not super-happy with this approach, as it mirrors what is already configured on the cobra command.

An alternative could be to create a `NewSkaffoldCommand` and filter valid arguments based on the cobra configuration. If this makes sense this should be done in a follow-up PR.

Reply:
How about using the cobra command to determine directly if the command has the given flags? Something like: 

```
c := cmd.NewSkaffoldCommand(os.Stdout, os.Stderr)
	var command *cobra.Command 
	for _, comm := range c.Commands() {
		if comm.Name() == b.command {
			command = comm 
		}
	}		
	if b.ns != "" && command.Flags().Lookup("namespace") != nil {
		args = append(args, "--namespace", b.ns)
	}
```

Reply:
Yes, this is what I was thinking too. Then I'll add it in this PR.